sovtokenfees/sovtokenfees/test/catchup/revert/test_revert_fees.py

```diff
@@ -0,0 +1,155 @@
+import functools
+
+import pytest
+
+from sovtoken.constants import XFER_PUBLIC, ADDRESS, AMOUNT, SEQNO, OUTPUTS
+from sovtokenfees.constants import FEES
+from sovtokenfees.test.helper import check_state, ensure_all_nodes_have_same_data, send_and_check_nym_with_fees
+
+from indy_common.constants import NYM, ATTRIB
+from plenum.common.startable import Mode
+from plenum.common.txn_util import get_seq_no
+from plenum.test.delayers import cDelay
+from plenum.test.helper import assertExp
+from plenum.test.stasher import delay_rules
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import ensure_view_change
+from stp_core.loop.eventually import eventually
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_max_size = tconf.Max3PCBatchSize
+    old_time = tconf.Max3PCBatchWait
+    tconf.Max3PCBatchSize = 2
+    tconf.Max3PCBatchWait = 5
```

Comment:
Why is it `5` and not `1000`? If we want to have a deterministic test to make sure that every batch has exactly 2 txns, then we must set `Max3PCBatchSize=2` and `Max3PCBatchWait ->infinity`.